alodataset/coco_detection_dataset.py

```diff
@@ -88,13 +92,18 @@ def __init__(
             dataset_dir = BaseDataset.get_dataset_dir(self)
             img_folder = os.path.join(dataset_dir, img_folder)
             ann_file = os.path.join(dataset_dir, ann_file)
+            stuff_ann_file = None if stuff_ann_file is None else os.path.join(dataset_dir, stuff_ann_file)
```

Comment:
Does the original detr paper use the panoptic ann file or the stuff file ? Using the stuff ann combined with the things ann file is an option. But we might considere using the panoptic file as well to match the original DETR implementation.

Reply:
I will create a new issue to make this